cloud-sql/mysql/mysql/server.js

```diff
@@ -99,34 +99,25 @@ app.on('listening', async () => {
 // Serve the index page, showing vote tallies.
 app.get('/', async (req, res) => {
   // Get the 5 most recent votes.
-  const recentResultPromise = pool
-    .query(
-      'SELECT candidate, time_cast FROM votes ORDER BY time_cast DESC LIMIT 5'
-    )
-    .then(rows => {
-      return rows;
-    });
+  const recentVotes = await pool.query(
```

Comment:
These needed to remain awaited below for all 3 to remain concurrent.

Reply:
Aren't they already?

(If you're referring to the `.count` property - I don't believe non-`function` properties can return promises.)

Reply:
Currently the sample does `Create, Create, Create -> Await, Await, Await` which means each statement is executing concurrently. This change moves it to a `Create -> Await, Create -> Await, Create -> Await` for each execution, which means each statement is performed serially. (I might be wrong since I'm not super familiar with Node, but that's my understanding of Async/Await)

Reply:
Ah, I see. Will fix. 🙂